lua/hop/window.lua

```diff
@@ -65,11 +65,12 @@ function M.get_window_context(multi_windows)
   end
 
   for _, w in ipairs(vim.api.nvim_tabpage_list_wins(0)) do
-    local b = vim.api.nvim_win_get_buf(w)
-    if w ~= cur_hwin then
+    if w ~= cur_hwin and vim.api.nvim_win_is_valid(w) then
+      local b = vim.api.nvim_win_get_buf(w)
+      if not vim.tbl_contains(excluded_filetypes, vim.api.nvim_buf_get_option(b, 'filetype')) then
```

Comment:
Please indent the following lines correctly.

Reply:
Sorry for my careless. I'll fix.